packages/vaadin-icon/theme/lumo/vaadin-icon-styles.js

```diff
@@ -0,0 +1,13 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
```

Comment:
Should import `'@vaadin/vaadin-lumo-styles/sizing.js';`

Reply:
Done.